dwml-data-parsers/parameter.js

```diff
@@ -72,9 +72,23 @@ var parameterParser = {
 
     while (i < values.length) {
       var currentValue = values[i];
+
+      currentTimeFrame = timeFrames[timeFrameCounter];
```

Comment:
Yeah, I ran into some issues moving things around in the loop. 

I was am going to try to clean up this loop a bit after this PR (didn't want to change too much at once)